maestro/README.md

```diff
@@ -68,28 +68,46 @@ metadata:
   labels:
     app: mas-example
 spec:
-  strategy:
-    type: sequence
-    output: verbose
   template:
     metadata:
       labels:
         app: mas-example
     agents:
+      - current-affairs
+      - hot-or-not
+    prompt: New York City
+    steps:
       - name: current-affairs
+        agent: current-affairs
       - name: hot-or-not
-    prompt: New York
+        agent: hot-or-not
 ```
 
+* Before running, validate the agent and workflow structure:
+
+```bash
+maestro validate agent_schema.json agent.yaml
```

Comment:
note for @maximilien : in the future, should we make it so the user doesn't need to pass the schema and instead could do something like `maestro validate --agent agent.yaml` ?